projects/core/src/checkout/store/effects/payment-types.effect.ts

```diff
@@ -31,6 +31,33 @@ export class PaymentTypesEffects {
     })
   );
 
+  @Effect()
+  setPaymentType$: Observable<
+    CheckoutActions.SetPaymentTypeSuccess | CheckoutActions.SetPaymentTypeFail
+  > = this.actions$.pipe(
+    ofType(CheckoutActions.SET_PAYMENT_TYPE),
+    map((action: any) => action.payload),
```

Comment:
Can we add a type for the action instead of any?  And perhaps a type for the payload, although I see we don't always do that win other effects.  Your call.

Reply:
added type for action.